sql-schema/183.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE users_prefs DROP PRIMARY KEY;
+ALTER TABLE users_prefs DROP INDEX pref;
+INSERT INTO `users_prefs` (`user_id`, `pref`, `value`) SELECT `user_id`, 'dashboard', `dashboard` FROM `users` WHERE `dashboard` != '0';
+INSERT INTO `users_prefs` (`user_id`, `pref`, `value`) SELECT `user_id`, 'twofactor', `twofactor` FROM `users` WHERE `twofactor` != '0';
+ALTER TABLE `users` DROP `twofactor`, DROP `dashboard`;
```

Comment:
This doesn't run:

`ERROR 1067 (42000): Invalid default value for 'updated_at'`

That is with `ONLY_FULL_GROUP_BY,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION` set in mysql but I've been through before and resolved all of those.

`1970-01-01 00:00:01` is what the default should be so will need that updating in the schema before the final ALTER TABLE line.

This has shown me that the db unit tests are broke. When you run `DBTEST=true phpunit tests/DBSetupTest.php` it's fine but if you just run `DBTEST=true phpunit` then it fails as it can't select the database on init as it's not been created yet. I'm not sure how to fix that other than try and create the db on initi if we can't switch.